src/functions.php

```diff
@@ -1,17 +1,24 @@
 <?php declare(strict_types=1);
 
-use Bref\Application;
+use Bref\Runtime\LambdaRuntime;
 
 /**
  * Shortcut for creating and running a simple lambda application.
  *
  * @param callable $handler This callable takes a $event parameter (array) and must return anything serializable to JSON.
- *
- * @see \Bref\Application::simpleHandler()
  */
 function λ(callable $handler): void
 {
-    $app = new Application;
-    $app->simpleHandler($handler);
-    $app->run();
+    lambda($handler);
```

Comment:
this will be a very common used function.

Would it be worth to duplicate the impl to save a function call?

Reply:
That made me think more about it and I opened #124 to actually remove this alias.